scripts/actions/mobskills/questionmarks_needles.lua

```diff
@@ -9,15 +11,24 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    -- from http://ffxiclopedia.wikia.com/wiki/%3F%3F%3F_Needles
-    -- "Seen totals ranging from 15, 000 to 55, 000 needles."
-    local needles = math.random(15000, 55000) / skill:getTotalTargets()
+        -- from http://ffxiclopedia.wikia.com/wiki/%3F%3F%3F_Needles
```

Comment:
should probably be under the `if` just before the variable holding the `math.random()` call. That saves re-aligning it since where it is now its indent is off.